tools/make/ffi.toml

```diff
@@ -68,7 +68,7 @@ description = "Build ICU4X CAPI for Cortex"
 category = "ICU4X FFI"
 dependencies = ["install-nightly", "install-cortex-8"]
 toolchain = "${ICU4X_NIGHTLY_TOOLCHAIN}"
-env = { RUSTFLAGS = "-Ctarget-cpu=cortex-m33 -Cpanic=abort -Copt-level=s" }
```

Comment:
question: why remove this?

Reply:
I got an error locally saying that the cpu is unknown